temoa-uiuc/data_files/run_mga_scenario3.txt

```diff
@@ -0,0 +1,45 @@
+#-----------------------------------------------------
+# This is a sample configuration file for Temoa
+# It allows you to specify (and document) all run-time model options
+# Legal chars in path: a-z A-Z 0-9 - _ \ / . :
+# Comment out non-mandatory options to omit them
+#-----------------------------------------------------
+
+# Input File (Mandatory)
+# Input can be a .sqlite or .dat file
+# Both relative path and absolute path are accepted
+--input=data_files/03_uiuc_mga.sqlite
```

Comment:
Are the input and output files supposed to be the same?

Reply:
Yes. I don't actually use the sqlite file output but it's required for the model to run